visualizer/visualize-dependencies.js

```diff
@@ -0,0 +1,92 @@
+const fs = require('fs');
```

Comment:
This can be turned into a self-contained shell script (rationale see https://github.com/CirclesUBI/land-local/pull/3/files#r1112888137) by applying

```suggestion
#!/usr/bin/env node
const fs = require('fs');
```